src/Halogen/HTML/Properties.purs

```diff
@@ -95,6 +95,7 @@ import Unsafe.Coerce (unsafeCoerce)
 newtype IProp (r :: # Type) i = IProp (Prop (InputF Unit i))
 
 derive instance newtypeIProp :: Newtype (IProp r i) _
+derive instance functorIProp :: Functor (IProp r)
```

Comment:
You could `derive newtype instance` here even I think?

Reply:
I don’t think so, `IProp r` is more like `Compose Prop (InputF Unit)`, but that’s not the newtype we have.

Reply:
In particular the implementation of it would use `map` twice: https://github.com/MonoidMusician/purescript-halogen-behaviors/blob/master/src/Halogen/Behavior/Internal.purs#L21 (from my project)